js/controllers/asteroids.js

```diff
@@ -1,6 +1,10 @@
 var canvas = new Canvas("game-canvas");
 var spaceship = new Spaceship();
-var asteroid = new Asteroid(100, 100, 0, 0, 3);
+
+var shpSize = randomNumber(1, 3);
```

Comment:
Calling a function from the model here could be dangerous, especially since the model is not included in this file.